<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndirectFitDataForm</class>
 <widget class="QTabWidget" name="IndirectFitDataForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>313</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::North</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <property name="elideMode">
   <enum>Qt::ElideLeft</enum>
  </property>
  <widget class="QWidget" name="loSingleInput">
   <attribute name="title">
    <string>Single Input</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbSample">
      <property name="text">
       <string>Sample</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="MantidQt::MantidWidgets::DataSelector" name="dsSample" native="true">
      <property name="autoLoad" stdset="0">
       <bool>true</bool>
      </property>
      <property name="ShowGroups" stdset="0">
       <bool>false</bool>
      </property>
      <property name="showLoad" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbResolution">
      <property name="text">
       <string>Resolution</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QWidget" name="wgtStartEnd" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Start X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsbStartX">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>End X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsbEndX">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-1000000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="MantidQt::MantidWidgets::DataSelector" name="dsResolution" native="true">
      <property name="autoLoad" stdset="0">
       <bool>true</bool>
      </property>
      <property name="ShowGroups" stdset="0">
       <bool>false</bool>
      </property>
      <property name="showLoad" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Multiple Input</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="tbFitData">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="loDataControl">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbAdd">
        <property name="text">
         <string>Add Workspace</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbRemove">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/DataSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
